### PR TITLE
Added san storage attribute

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Added san storage attribute]
### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: oneview/resource_server_profile.go [hunk 1]
```diff
@@ -1416,19 +1454,14 @@ func resourceServerProfileCreate(d *schema.ResourceData, meta interface{}) error
...
-				URI:                   utils.NewNstring(sanStorageItem["uri"].(string)),
+				HostOSType:       sanStorageItem["host_os_type"].(string),
+				ManageSanStorage: sanStorageItem["manage_san_storage"].(bool),
 			}
 		}
 		serverProfile.SanStorage = sanStorage
 	}
 	if _, ok := d.GetOk("volume_attachments"); ok {
-		rawVolumeAttachments := d.Get("volume_attachments").(*schema.Set).List()
+		rawVolumeAttachments := d.Get("volume_attachments").([]interface{})
```
Comment (VenkateshRavula) [inline oneview/resource_server_profile.go:1464]: what is the difference b/w ```interface``` and ```schema.Set```

**Review by VenkateshRavula — APPROVED** [review]

**Review by AsisBagga — COMMENTED** [review]
Please Update docs.

**Comment by nabhajit-ray** [comment]
> Please Update docs.

Done

File: oneview/resource_server_profile.go (hunk 1 above)
Comment (nabhajit-ray) [reply oneview/resource_server_profile.go:1464]: We change the  type from typeset to typelist so the conversion is added

**Review by AsisBagga — APPROVED** [review]
LGTM

**Pull request merged**